SushiRunner.Data/Repositories/CommentRepository.cs

```diff
@@ -0,0 +1,93 @@
+using System;
```

Comment:
No need for CommentRepository. If you want to add a comment to a meal you just do this:

```dotnet
var meal = _mealService.Get(mealId);
meal.Comments.Add(comment);
```